ui/lib/core/addon/templates/components/replication-secondary-card.hbs

```diff
@@ -0,0 +1,65 @@
+<div class="selectable-card is-rounded card-container">
+  <h3 class="title is-4 grid-item-title card-title">{{title}}</h3>
+  <div class="grid-item-state">
+    {{#if property_3}}
+      <code class="has-text-black">{{property_3}}</code>
+      <p class="has-text-grey is-size-8">{{description_3}}</p>
+    {{else}}
+      <code class="has-text-black">{{property_1}}</code>
+      <p class="has-text-grey is-size-8">{{description_1}}</p>
+    {{/if}}
+  </div>
+  <div class="grid-item-wal">
```

Comment:
is this class name here and in the other divs correct or should it be dynamically set? i.e. will this item always be the WAL? it looks like it might also be the state, right?

Reply:
I cleaned this up as well.